dist/pythonlibs/testrunner/__init__.py

```diff
@@ -1,57 +1,21 @@
-# Copyright (C) 2017 Cenk Gündoğan <cenk.guendogan@haw-hamburg.de>
+# Copyright (C) 2018-19 Freie Universität Berlin
+#               2017 Cenk Gündoğan <cenk.guendogan@haw-hamburg.de>
 #               2016 Kaspar Schleiser <kaspar@schleiser.de>
 #               2014 Martine Lenders <mlenders@inf.fu-berlin.de>
 #
 # This file is subject to the terms and conditions of the GNU Lesser
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-import os
-import signal
 import sys
-import subprocess
-import time
-from traceback import extract_tb, print_tb
-
+from traceback import print_tb
```

Comment:
Unused import.

Reply:
Untrue. It's used here https://github.com/RIOT-OS/RIOT/blob/ac50244f85773477e85b495eb645d92f5a12c579/dist/pythonlibs/testrunner/__init__.py#L33

Reply:
It is not however used in `spawn.py`. Removed it there.

Reply:
Sorry I mixed the files yes.